persistence/jdo/datanucleus/src/main/java/org/apache/isis/persistence/jdo/datanucleus/metamodel/facets/entity/JdoEntityFacet.java

```diff
@@ -80,6 +81,7 @@
     @Inject private ObjectManager objectManager;
     @Inject private ExceptionRecognizerService exceptionRecognizerService;
     @Inject private JdoFacetContext jdoFacetContext;
+    @Inject private List<IdStringifier> idStringifiers;
```

Comment:
might be irrelevant, but probably use
```java
@Autowired(required=false) private List<IdStringifier> idStringifiers;
```
just to be safe when JUnit testing, and no IdStringifiers are registered

Reply:
No longer necessary, there will always be `IdStringifier`s available.